pkg/daemon/containerrecreate/crr_daemon_controller.go

```diff
@@ -362,10 +363,25 @@ func (c *Controller) manage(crr *appsv1alpha1.ContainerRecreateRequest) error {
 			}
 			return c.patchCRRContainerRecreateStates(crr, newCRRContainerRecreateStates)
 		}
+
+		if crr.Spec.Strategy.ForceRecreate {
+			newForceKilledContainerStatuses = append(newForceKilledContainerStatuses, appsv1alpha1.ContainerRecreateRequestForceKilledContainerStatus{
```

Comment:
we don't have to patch "forcekill" status if the container do not have to be recreated forcefully

Reply:
if crr.Spec.Strategy.ForceRecreate && state.ShouldBeKilledByForce {
			state.IsForceKilled = true
			state.ContainerID = kubeContainerStatus.ID.String()
		}